about.html

```diff
@@ -12,53 +12,106 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
+<div class = "container">
+    <div class="navbar">
+                
+    <img src="C:\Users\Andrew\Desktop\User-Interface\Sprint-Challenge--User-Interface\img\lambda-black.png" alt="strategy" class="logo">
+    
+      <nav class="headnav">
+          <a href="C:\Users\Andrew\Desktop\User-Interface\Sprint-Challenge--User-Interface\index.html">Home</a>
+          <a href="#">About</a>
+          <a href="#">Products</a>
+          <a href="#">Blog</a>
+          <a href="#">Contact</a>  
+      </nav>
+        </div>
+    
+    <img src = "C:\Users\Andrew\Desktop\User-Interface\Sprint-Challenge--User-Interface\img\jumbo-about.png" alt = "strategy">
```

Comment:
This is incorrect image source syntax. When writing the source path for images, **never** set this to a local directory on your machine. Rather, set the image source in relation to where it exists compared to your main directory (usually the one containing index.html). Instead of
`src="C:\Users\Andrew\Desktop\User-Interface\Sprint-Challenge--User-Interface\img\jumbo-about.png"`
we would instead want to see
`src="/img/jumbo-about.png"`